### DISCUSSION TITLE
Hololens 1 vs Emulator vs Unity Editor MixedRealityKeyboard Example Behavior MRTK 2.4.0 or 2.5.3

### DISCUSSION THREAD

**Discussion by ThanosRestas** [microsoft/MixedRealityToolkit-Unity#9091] [Q&A]
Hello everyone.  

Current Setup :  **MRTK 2.4.0** , **Unity 2019.4.10f1**, **Hololens 1 Emulator**

The situation :

1. **Unity Editor** + **MixedRealityKeyboard Example scene** = Works perfectly
2. **Hololens 1 Emulator** + **MixedRealityKeyboard Example scene** = Input Field doesn't seem to work, can't be clicked, can't enter text.
3.  **Hololens 1 Device**  + **MixedRealityKeyboard Example scene** = ????

I don't have access to a Hololens 1 Device at the moment and  I would like to know if the MixedRealityKeyboard Example scene works properly when using the actual device just for a sanity check.

Additional affirmations on the incorrect behavior of the scene when using the Hololens 1 Emulator will be deeply appreciated too to clear that it's not on my end only.

Thanks !

**Comment by keveleigh** [comment]
@MaxWang-MS Do you have any insight into this behavior?

**Reply by MaxWang-MS** [reply]
Unfortunately I cannot run emulator on my machine for some reason and also do not have access to a HoloLens first gen device so I don't think I know the answer.
